<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerSelection</class>
 <widget class="QWidget" name="PlayerSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="player1SelectX">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);
border:none</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="player1SelectO">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>190</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);
border:none</string>
   </property>
   <property name="text">
    <string>o</string>
   </property>
  </widget>
  <widget class="QPushButton" name="player2SelectO">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);
border:none</string>
   </property>
   <property name="text">
    <string>O</string>
   </property>
  </widget>
  <widget class="QLabel" name="player2Label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>349</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gilroy-Bold</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="player1Label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>146</y>
     <width>611</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gilroy-Bold</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="player2SelectX">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>410</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);
border:none</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="player1Starts">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);
border:none</string>
   </property>
   <property name="text">
    <string>select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="player2Starts">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>450</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);
border:none</string>
   </property>
   <property name="text">
    <string>select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spSelectX">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>201</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spSelectO">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
</string>
   </property>
   <property name="text">
    <string>O</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spStartGame">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>460</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
</string>
   </property>
   <property name="text">
    <string>select</string>
   </property>
  </widget>
  <widget class="QWidget" name="multiplayerWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="singleplayerWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <zorder>singleplayerWidget</zorder>
  <zorder>player2Label</zorder>
  <zorder>player1Label</zorder>
  <zorder>multiplayerWidget</zorder>
  <zorder>spSelectX</zorder>
  <zorder>spSelectO</zorder>
  <zorder>spStartGame</zorder>
  <zorder>player1SelectO</zorder>
  <zorder>player1SelectX</zorder>
  <zorder>player1Starts</zorder>
  <zorder>player2SelectO</zorder>
  <zorder>player2SelectX</zorder>
  <zorder>player2Starts</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
